content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,250 @@
+    namespace: dev
+    verb: create
+status:
+  allowed: true
+  denied: false
+```
+
+## Usando flags para seu módulo de autorização
+
+Você deve incluir uma flag em sua política para indicar qual módulo de autorização
+suas políticas incluem:
+
+As seguintes flags podem ser utilizadas:
+
+  * `--authorization-mode=ABAC` O modo de controle de acesso baseado em atributos [Attribute-Based Access Control (ABAC)] permite configurar políticas usando arquivos locais.
+  * `--authorization-mode=RBAC` O modo de controle de acesso baseado em função [Role-based access control (RBAC)] permite que você crie e armazene políticas usando a API do Kubernetes.
+  * `--authorization-mode=Webhook` WebHook é um modo de retorno de chamada HTTP que permite gerenciar a autorização usando endpoint REST.
+  * `--authorization-mode=Node` A autorização de nó é um modo de autorização de propósito especial que autoriza especificamente requisições de API feitas por kubelets.
+  * `--authorization-mode=AlwaysDeny` Esta flag bloqueia todas as requisições. Utilize esta flag somente para testes.
+  * `--authorization-mode=AlwaysAllow` Esta flag permite todas as requisições. Utilize esta flag somente se não existam requisitos de autorização para as requisições de API.
+
+Você pode escolher mais de um modulo de autorização. Módulos são verificados
+em ordem, então, um modulo anterior tem maior prioridade para permitir ou negar uma requisição.
+
+## Escalonamento de privilégios através da criação ou edição da cargas de trabalho {#privilege-escalation-via-pod-creation}
+
+Usuários que podem criar ou editar pods em um namespace diretamente ou através de um [controlador](/pt-br/docs/concepts/architecture/controller/)
+como, por exemplo, um operador, e conseguiriam escalar seus próprios privilégios naquele namespace.
+
+{{< caution >}}
+Administradores de sistemas, tenham cuidado ao permitir acesso para criar ou editar cargas de trabalho.
+Detalhes de como estas permissões podem ser usadas de forma maliciosa podem ser encontradas em [caminhos para escalonamento](#escalation-paths).
+
+{{< /caution >}}
+
+### Caminhos para escalonamento {#escalation-paths}
+
+- Montar Secret arbitrários nesse namespace
+   - Pode ser utilizado para acessar Secret destinados a outras cargas de trabalho
+   - Pode ser utilizado para obter um token da conta de serviço com maior privilégio
+- Usando contas de serviço arbitrárias nesse namespace
+   - Pode executar ações da API do Kubernetes como outra carga de trabalho (personificação)
+   - Pode executar quaisquer ações privilegiadas que a conta de serviço tenha acesso
+- Montagem de configmaps destinados a outras cargas de trabalho nesse namespace
+   - Pode ser utilizado para obter informações destinadas a outras cargas de trabalho, como nomes de host de banco de dados.
+- Montar volumes destinados a outras cargas de trabalho nesse namespace
+   - Pode ser utilizado para obter informações destinadas a outras cargas de trabalho e alterá-las.
```

Comment:
For consistency, I think all "mounting" should be translated to "montagem" and "using" could be translated to "uso de" instead.